pandas/core/internals/managers.py

```diff
@@ -631,6 +631,13 @@ def replace_list(
         bm._consolidate_inplace()
         return bm
 
+    def to_native_types(self, **kwargs) -> "BlockManager":
```

Comment:
more informative name?

Reply:
It's the name that the blocks also use..

Reply:
It's not an informative name there either

Reply:
Then I am tempted to say "Go for it" .. ;)

I am well aware and fully agree that the whole "native types" terminology is confusing, but it is used in quite some places (not only the Block method, but also the Index helper methods). So fully on board with changing it, but that's a bigger change than just trying to remove the internals leakage in the csv code what I am doing here.

Reply:
yeah this can be updated / changed later